docs/getting-started/delta-lake-format.md

```diff
@@ -0,0 +1,154 @@
+We could register our CSV files in this catalog by creating a table via a DDL-statement, including the definition of all columns, path and format of our data.
+But you could also directly create and write into a table by using Spark Hive tables. 
+Smart Data Lake Builder supports this by the HiveTableDataObject. It always uses Parquet file format in the background as a best practice, although Hive tables could also be created on top of CSV files.
+
+:::info
+Hive is a Metadata layer and SQL engine on top of a Hadoop filesystem. Spark uses the metadata layer of Hive, but implements its own SQL engine.
+:::
+
+## Transactions
+
+Hive tables with Parquet format are lacking transactions. This means for example that writing and reading the table at the same time could result in failure or empty results. 
+In consequence 
+* consecutive jobs needs to by synchronized
+* it's not recommended having end-user accessing the table while data processing jobs are running
+* update and deletes are not supported
+
+There are other options like classical databases which always had a metadata layer, offer transactions but don't integrate easily with Hive metastore and cheap Hadoop file storage.
+Nevertheless, Smart Data Lake Builder supports classical databases through the JdbcTableDataObject.
+Fortunately there is a new technology called Delta Lake, see also [delta.io](https://delta.io/). It integrates in Hive metastore, supports transactions and stores Parquet files and a transaction log on hadoop filesystems.
+Smart Data Lake Builder supports this by the DeltaLakeTableDataObject, and this is what we are going to use for our airport and departure data now.
+
+## DeltaLakeTableDataObject
+
+Switching to Delta Lake format is easy with Smart Data Lake Builder, just replace CsvFileDataObject with DeltaLakeTableDataObject, and define the table's name:
+
+    int-airports {
+        type = DeltaLakeTableDataObject
+        path = "~{id}"
+        table = {
+            db = default
+            name = int_airports
+        }
+    }
+
+Then create a similar data object int-departure, and a new action prepare-departures to fill it:
+
+    int-departures {
+        type = DeltaLakeTableDataObject
+        path = "~{id}"
+        table = {
+            db = default
+            name = int_departures
+        }
+    }
+    
+    prepare-departures {
+        type = CopyAction
+        inputId = stg-departures
+        outputId = int-departures
+        metadata {
+            feed = compute
+        }
+    }
+
+Finally, adapt action join-departures-airports:
+* change stg-departures to int-departures in inputIds of action join-departures-airports
+* change stg_departures to int_departures in first SQLDfsTransformer
+
+To run our data pipeline, first delete the data directory - otherwise DeltaLakeTableDataObject will fail because of existing files in different format.
+Then you can execute the usual *docker run* command for all feeds:
```

Comment:
By following exactly these instructions, I get the error `IllegalArgumentException: requirement failed: (DataObject~stg-airports) DataObject schema is undefined. A schema must be defined if there are no existing files.`. If there are some people, like me, that forgot the part that the first time, you should run the download step alone, they may become frustrated with the tutorial.

I would suggest to either repeat here to first run the download selector, or, maybe better, have a warning block below this execution with the text: "Getting this error message? (bla bla): remember that when a file is not yet present, SDL can not validate the schema of the whole DAG before running"